packages/kit/src/provider/Container/InAppNotification/index.tsx

```
@@ -34,6 +57,178 @@ const InAppNotification = () => {
+    return (
+      <Button
+        variant="secondary"
+        size="small"
+        onPress={approvingSuccessActionConfirm}
+      >
+        <SizableText>
+          {intl.formatMessage({ id: ETranslations.swap_toast_go_to_swap })}
+        </SizableText>
+      </Button>
+    );
+  }, [approvingSuccessActionConfirm]);
```

ESLint

[error] React Hook useMemo has a missing dependency: 'intl'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)